ccswbs/hjckrrh
# Issue 167 - Make E6, PP4, F4 consistent with N7

Fixes #167.

Matched view settings for E6, PP4, and F4 with N7, which has an unformatted list. Copied N7 template files for E6, PP4, and F4 to achieve the comma-delimited keyword list. Renamed F4 view to "F4" for consistency.

## Test Procedure

Check an FAQ, people profile, event, and news item. For each, note the presentation under "Find related [content type] by keyword" when multiple keywords are used. Each keyword list should be comma-delimited:

![screenshot-localhost-2017-04-18-10-07-00](https://cloud.githubusercontent.com/assets/25161107/25135514/9519f9d4-2420-11e7-8cc0-c3a99dea109e.png)
